weblab/experiments/tests/test_views.py

```diff
@@ -248,59 +268,124 @@ def test_view_my_experiments_empty_for_anonymous(self, client, helpers, experime
         assert len(data['getMatrix']['experiments']) == 0
```

Comment:
Might be good if the test case had something to show here, actually, as a sanity check!